pyvista/plotting/colors.py

```diff
@@ -947,8 +947,9 @@ def get_cmap_safe(cmap):
     """Fetch a colormap by name from matplotlib, colorcet, or cmocean."""
     try:
         from matplotlib.cm import get_cmap
-    except ImportError:
-        raise ImportError('cmap requires matplotlib')
+    except ImportError:  # pragma: no cover
+        raise ImportError('The use of custom colormaps requires the installation of matplotlib')
```

Comment:
I would recommend `raise ImportError(...) from None` here and below to suppress the original exception context. Although "`ModuleNotFoundError: No module named 'matplotlib'` is clear enough, it might confuse end users less if we only show our own error message.

Otherwise LGTM too.

Reply:
```suggestion
    except ImportError:  # pragma: no cover
        raise ImportError('The use of custom colormaps requires the installation of matplotlib') from None
```